apps/web-giddh/src/app/theme/ng-virtual-select/sh-select-menu.component.ts

```diff
@@ -63,19 +64,21 @@ export class ShSelectMenuComponent implements OnChanges {
         if (changes['isRequired'] && changes['isRequired'].currentValue !== changes['isRequired'].previousValue) {
             this.autoSelectIfSingleValueAvailable();
         }
+
+        if(changes['isSelectAllChecked']) {
```

Comment:
Please check this as it will override the value of this input resulting in current value to be unreachable